scripts/superdesk-search/search.js

```diff
@@ -2167,7 +2167,7 @@
                     scope.save = function(editSearch) {
 
                         function onSuccess() {
-                            notify.success(gettext('Saved search is saved successfully'));
+                            notify.success(gettext('Search was saved successfully'));
```

Comment:
Having looked through the strings again, apart from a few instances of "There is an error", "Password is changed" and "User is not found", we generally use the past tense for notifications. I'll add a commit to make these consistent.